### ISSUE TITLE
How can I use tokenizer.encode_plus to input and encode 2 sentences - (query,answer) pair for training a BERT binary classifier?

### ISSUE THREAD

**Issue by soumya-ranjan-sahoo** [huggingface/transformers#5041]
# ❓ Questions & Help

## Details

**A link to original question on Stack Overflow**:

**Comment by soumya-ranjan-sahoo** [comment]
Here my intention is to train a BERT binary classifier which classifies if an answer corresponding to a given query is correct. How do I proceed to encode the query, answer pair in the Input?

**Comment by bright1993ff66** [comment]
@soumya-ranjan-sahoo You could check the [token type ids](https://huggingface.co/transformers/glossary.html#token-type-ids). Please feed the question and answer to the, for instance, encode_plus function and generate type type ids. 

Then you could just feed the token type ids with, for instance, input ids and attention masks to conduct classification

**Comment by soumya-ranjan-sahoo** [comment]
@bright1993ff66 Great. I was successful in my experiment. But now I have a follow-up question. I understand the maximum length of the permissible words for BERT is 512. In my case (sentence pair classification) does that imply the combined word length for the query and the answer has to be 512 since I have huge answers for my experiment. 
Surprisingly I was able to fine-tune BERT with query and answer (most query-answer pair have a combined word length of more than 512), and BERT didn't throw any error or warnings. How did it fine-tune or what it exactly did with such sentences? 

Thank you.
